client/src/main/java/com/battleships/start_window/InsertData/InsertSeekBarData.java

```diff
@@ -0,0 +1,40 @@
+package com.battleships.start_window.InsertData;
+
+import com.battleships.Translator;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+
+public class InsertSeekBarData {
```

Comment:
Name of class sounds like method name
SeekBarDataInserter?

Reply:
done